OpenHardwareMonitorLib/Hardware/HDD/WindowsNVMeSmart.cs

```diff
@@ -81,6 +81,7 @@ internal abstract class NVMeHealthInfo {
   internal class WindowsNVMeSmart : IDisposable {
     private enum Command : uint {
       IoctlScsiMiniPort = 0x04d008,
```

Comment:
is the commit message still true ? You want to merge this?

Reply:
Thanks for finding. Fixed.